src/components/Items/ProfileContact.js

```diff
@@ -56,15 +56,16 @@
             {fullname} {handleYouText()}
           </Text>
         </View>
-        {showArrow && (
-          <>
-            {userId !== item.user_id && (
-              <View>
-                <Mi name="arrow-forward-ios" size={18} />
-              </View>
-            )}
-          </>
-        )}
+        {showArrow &&
+          !(isYou || anonProfileId === item?.user_id || signedProfileId === item?.user_id) && (
+            <>
+              {userId !== item.user_id && (
+                <View>
+                  <Mi name="arrow-forward-ios" size={18} />
+                </View>
+              )}
+            </>
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new condition for showing the arrow seems more complex than before. It now checks not only if the user is the current user (`isYou`) but also if the user is an anonymous user or a signed user. This change seems reasonable considering the introduction of anonymous users. However, please make sure that all these conditions are necessary and correctly handle all possible scenarios.
```diff
-        {showArrow && (
+        {showArrow &&
+          !(isYou || anonProfileId === item?.user_id || signedProfileId === item?.user_id) && (
           <>
             {userId !== item.user_id && (
               <View>
                 <Mi name="arrow-forward-ios" size={18} />
               </View>
             )}
           </>
         )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->